### PR TITLE
Add getbit Spark function

### PR DESCRIPTION
**Pull request by acvictor**
Add `getbit` Spark function which is an [alias](https://github.com/apache/spark/blob/7b1147a05a6ca54276538d766c089980b9ee5d59/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala#L807) for `bit_get`.

Spark function doc - https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.functions.getbit.html#

### CODE REVIEW

**Comment by acvictor** [comment]
@PHILO-HE can you please review? Thank you!

**Comment by acvictor** [comment]
@bikramSingh91 can you please review this? Thank you!

**Review by PHILO-HE — COMMENTED** [review]
Looks good! cc @mbasmanova

File: velox/docs/functions/spark/bitwise.rst [hunk 1]
```diff
@@ -41,6 +41,10 @@ Bitwise Functions
     The value of 'pos' argument must be between 0 and number of bits in 'x' - 1.
     Invalid 'pos' values result in an error.
 
+.. spark:function:: getbit(x, pos) -> tinyint
+
+    This is an alias for ``bit_get(x, pos)``.
```
Comment (mbasmanova) [inline velox/docs/functions/spark/bitwise.rst:46]: nit: drop 'This is an' and add a link to bit_get.

```
Alias for :spark:func:`bit_get`
```

**Review by mbasmanova — APPROVED** [review]
@acvictor Looks good % one nit.

**Comment by zhli1142015** [comment]
I think we may not need to handle Spark function alias, as Gluten maps the Spark function to velox function by class name not registered name.
Thanks.

**Comment by PHILO-HE** [comment]
> I think we may not need to handle Spark function alias, as Gluten maps the Spark function to velox function by class name not registered name. Thanks.

Yes, Spark creates `BitwiseGet` class for this function, and in Gluten it is mapped to a unique function name "bit_get" (see [link](https://github.com/search?q=repo%3Aapache%2Fincubator-gluten+BITWISE_GET+&type=code)). As "bit_get" is already the registered function name. Maybe, there is no need to register the alias name in Velox.

**Comment by acvictor** [comment]
> > I think we may not need to handle Spark function alias, as Gluten maps the Spark function to velox function by class name not registered name. Thanks.
> 
> Yes, Spark creates `BitwiseGet` class for this function, and in Gluten it is mapped to a unique function name "bit_get" (see [link](https://github.com/search?q=repo%3Aapache%2Fincubator-gluten+BITWISE_GET+&type=code)). As "bit_get" is already the registered function name. Maybe, there is no need to register the alias name in Velox. @acvictor, could you confirm this? Thanks!

Thanks @zhli1142015 @PHILO-HE for the suggestion! I will test by adding the mapping in Gluten and revert.

**Comment by PHILO-HE** [comment]
> > > I think we may not need to handle Spark function alias, as Gluten maps the Spark function to velox function by class name not registered name. Thanks.
> > 
> > 
> > Yes, Spark creates `BitwiseGet` class for this function, and in Gluten it is mapped to a unique function name "bit_get" (see [link](https://github.com/search?q=repo%3Aapache%2Fincubator-gluten+BITWISE_GET+&type=code)). As "bit_get" is already the registered function name. Maybe, there is no need to register the alias name in Velox. @acvictor, could you confirm this? Thanks!
> 
> Thanks @zhli1142015 @PHILO-HE for the suggestion! I will test by adding the mapping in Gluten and revert.

@acvictor, I guess we don't need any code change in Gluten. You can just try to test with the alias function name, then check whether it is offloaded to Velox. E.g.,

`SELECT getbit(c0, 0) from t;`

**Comment by acvictor** [comment]
> > > > I think we may not need to handle Spark function alias, as Gluten maps the Spark function to velox function by class name not registered name. Thanks.
> > > 
> > > 
> > > Yes, Spark creates `BitwiseGet` class for this function, and in Gluten it is mapped to a unique function name "bit_get" (see [link](https://github.com/search?q=repo%3Aapache%2Fincubator-gluten+BITWISE_GET+&type=code)). As "bit_get" is already the registered function name. Maybe, there is no need to register the alias name in Velox. @acvictor, could you confirm this? Thanks!
> > 
> > 
> > Thanks @zhli1142015 @PHILO-HE for the suggestion! I will test by adding the mapping in Gluten and revert.
> 
> @acvictor, I guess we don't need any code change in Gluten. You can just try to test with the alias function name, then check whether it is offloaded to Velox. E.g.,
> 
> `SELECT getbit(c0, 0) from t;`

You're right, this works - https://github.com/apache/incubator-gluten/pull/5633

**Pull request closed without merging**